docs/UserGuide.md

```diff
@@ -177,7 +177,7 @@ This section provides information about all commands that is available in TutorB
 This section details the format of the commands available in TutorBuddy. We will adhere to the following:
 
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
+* Words in `UPPER_CASE` are the parameters / information to be supplied by the user.<br>
```

Comment:
Should we just put this in the glossary haha. I feel like she cannot really mark us down if we put it at the bottom

Reply:
Haha yeah, but honestly, I was thinking how we could explain parameters. The example given in the next line actually describes parameter quite well for me. So I am not sure what else we could add differently in the glossary, so in the end I didn't ah. Do let me know if you have any ideas on how we could describe this better in the glossary ah